communitys_franken.json

```diff
@@ -251,7 +271,10 @@
 			"nodeMaps" : [{
 				"url" : "https://monitoring.freifunk-franken.de/map?mapcenter=49.13304,10.07652,14",
 				"mapType" : "geographical"
-			}]
+			}],
+			"socialprojects": {
+				"number": 2
+			},
```

Comment:
This comma breaks the JSON.
Did you test or validate before creating this pull request?